code/drasil-printers/Language/Drasil/DOT/SWHS_TG.dot

```diff
@@ -0,0 +1,69 @@
+digraph SWHS_TG {
```

Comment:
Generated files should not be in the repo (exception is 'stable' where these are marked as 'golden' versions).